vllm/attention/backends/torch_sdpa.py

```diff
@@ -138,14 +138,14 @@ def __init__(
                 "Please use xFormers backend instead.")
 
     def forward(
-        self,
-        query: torch.Tensor,
-        key: torch.Tensor,
-        value: torch.Tensor,
-        kv_cache: Optional[torch.Tensor],
-        attn_metadata: TorchSDPAMetadata,  # type: ignore
-        kv_scale: float = 1.0,
-    ) -> torch.Tensor:
+            self,
+            query: torch.Tensor,
+            key: torch.Tensor,
+            value: torch.Tensor,
+            kv_cache: Optional[torch.Tensor],
+            attn_metadata: TorchSDPAMetadata,  # type: ignore
+            kv_scale: float = 1.0,
+            attn_type: AttentionType = AttentionType.DECODER) -> torch.Tensor:
```

Comment:
nit: This will reduce the number of line changes and make the code look better.
```suggestion
            attn_type: AttentionType = AttentionType.DECODER,
            ) -> torch.Tensor:
```